tests/core/tests/authentication.py

```diff
@@ -355,6 +355,7 @@ def test_check_active_true(self):
 
 
 @skipIf(not oauth2 or not oauth_provider, "oauth provider not installed")
+@skipIf(settings.DJANGO_VERSION >= settings.DJANGO_11, 'oauth-plus not compatible with django 1.11')
```

Comment:
oauth_provider / oauth-plus hasn't been updated in a while.  there are forks that have been, and we might need to point users at those or update the test runner to install one of those instead.  maybe fork one of those forks into the django-tastypie org to ensure it stays put?